src/new-client/src/components/headerActionStyles.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components"
+
+export const HeaderTearOutAction = styled.button`
```

Comment:
If this is only used by the `TearOutComponent` then a better place would be in `TearOutComponent.styles.ts` in the same folder